source/tests/system/nirfmxwlan_driver_api_tests.cpp

```diff
@@ -650,34 +632,34 @@ TEST_F(NiRFmxWLANDriverApiTests, OFDMModAccMIMOFromExample_FetchData_DataLooksRe
-  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_stream_rmsevm_per_subcarrier_mean_trace_response.x0());
-  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_stream_rmsevm_per_subcarrier_mean_trace_response.dx());
-  EXPECT_EQ(0, ofdm_mod_acc_fetch_stream_rmsevm_per_subcarrier_mean_trace_response.stream_rms_evm_per_subcarrier_mean_size());
-  EXPECT_EQ(0, ofdm_mod_acc_fetch_stream_rmsevm_per_subcarrier_mean_trace_response.stream_rms_evm_per_subcarrier_mean().size());
-  // EXPECT_EQ(0.0, ofdm_mod_acc_fetch_stream_rmsevm_per_subcarrier_mean_trace_response.stream_rms_evm_per_subcarrier_mean(0));
-  EXPECT_EQ(0, ofdm_mod_acc_fetch_pilot_constellation_trace_response.pilot_constellation().size());
-  // EXPECT_EQ(0.0, ofdm_mod_acc_fetch_pilot_constellation_trace_response.pilot_constellation(0).real());
-  // EXPECT_EQ(0.0, ofdm_mod_acc_fetch_pilot_constellation_trace_response.pilot_constellation(0).imaginary());
-  EXPECT_EQ(0, ofdm_mod_acc_fetch_data_constellation_trace_response.data_constellation().size());
-  // EXPECT_EQ(0.0, ofdm_mod_acc_fetch_data_constellation_trace_response.data_constellation(0).real());
-  // EXPECT_EQ(0.0, ofdm_mod_acc_fetch_data_constellation_trace_response.data_constellation(0).imaginary());
-  EXPECT_NE(0.0, ofdm_mod_acc_fetch_cross_power_response.cross_power_mean());
-  EXPECT_GT(0.0, ofdm_mod_acc_fetch_iq_impairments_response.relative_iq_origin_offset_mean());
-  EXPECT_LT(0.0, ofdm_mod_acc_fetch_iq_impairments_response.iq_gain_imbalance_mean());
-  EXPECT_GT(0.0, ofdm_mod_acc_fetch_iq_impairments_response.iq_quadrature_error_mean());
-  EXPECT_NE(0.0, ofdm_mod_acc_fetch_iq_impairments_response.absolute_iq_origin_offset_mean());
-  EXPECT_NE(0.0, ofdm_mod_acc_fetch_iq_impairments_response.iq_timing_skew_mean());
+  EXPECT_LE(number_of_users, number_of_stream_results);
+  EXPECT_GT(0.0, ofdm_mod_acc_fetch_frequency_error_mean_response_vec[0].frequency_error_mean());
+  EXPECT_GT(0.0, ofdm_mod_acc_fetch_symbol_clock_error_mean_response_vec[0].symbol_clock_error_mean());
+  EXPECT_LT(0.0, ofdm_mod_acc_fetch_stream_rmsevm_response_vec[0][0].stream_rms_evm_mean());
+  EXPECT_LT(0.0, ofdm_mod_acc_fetch_stream_rmsevm_response_vec[0][0].stream_data_rms_evm_mean());
+  EXPECT_LT(0.0, ofdm_mod_acc_fetch_stream_rmsevm_response_vec[0][0].stream_pilot_rms_evm_mean());
+  EXPECT_EQ(-28.0, ofdm_mod_acc_fetch_stream_rmsevm_per_subcarrier_mean_trace_response_vec[0][0].x0());
+  EXPECT_EQ(1.0, ofdm_mod_acc_fetch_stream_rmsevm_per_subcarrier_mean_trace_response_vec[0][0].dx());
+  EXPECT_EQ(57, ofdm_mod_acc_fetch_stream_rmsevm_per_subcarrier_mean_trace_response_vec[0][0].stream_rms_evm_per_subcarrier_mean_size());
+  EXPECT_EQ(57, ofdm_mod_acc_fetch_stream_rmsevm_per_subcarrier_mean_trace_response_vec[0][0].stream_rms_evm_per_subcarrier_mean().size());
+  // EXPECT_EQ(0.0, ofdm_mod_acc_fetch_stream_rmsevm_per_subcarrier_mean_trace_response_vec[0][0].stream_rms_evm_per_subcarrier_mean(0));
```

Comment:
Did something change to make these non-zero? Either way, if they are consistently populating now can we uncomment the assertions on the first values?

Reply:
It's because we're now looking at the _first_ values instead of the _last_. But good point, I'll go uncomment those.